src/common/io.ts

```diff
@@ -203,7 +207,7 @@ export class ARIOReadable implements AoARIORead {
   public readonly process: AOProcess;
   protected epochSettings: AoEpochSettings | undefined;
   protected arweave: Arweave;
-  protected paymentProvider: TurboArNSPaymentProvider; // TODO: this could be an array/map of payment providers
+  protected paymentProvider: TurboArNSPaymentProviderUnauthenticated; // TODO: this could be an array/map of payment providers
```

Comment:
see other comment - modify this to be Record<string, PaymentProvider>; and access it via `fundFrom`